### PR TITLE
Added generalised response

### LINKED ISSUE #846: Modify Response Class
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behaviour:**
In https://github.com/fossasia/badgeyay/blob/development/backend/blueprint/api/utils/response.py#L12:L16, If I want to pass general response like "User created successfully", other fields will also appear like id, attributes which are not needed. 

**Expected behaviour:**
Response method should respond to the type of messages like for general message it should only include a message and status

**I want to work on this issue**

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Fixes #846 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
- Added generalized response

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
Cc @djmgit @gabru-md @ParthS007 @yashLadha Please review

File: backend/blueprint/api/controllers/assetHelper.py [hunk 1]
```diff
@@ -26,4 +26,4 @@ def getDefaultBackgrounds():
             }
             resp.append(obj)
     resp_ = {'data': resp}
-    return jsonify(resp_)
+    return jsonify(Response(200).generateMessage(resp_, 'Image Response'))
```
Comment (yashLadha) [inline backend/blueprint/api/controllers/assetHelper.py]: This won't work as ember relies on json api convention data or rest data (for which each key should have a different model). Please go through the ember docs at least once.

**Comment by yashLadha** [comment]
Please also read it https://gitter.im/fossasia/badgeyay?at=5b08c985c5750d377c1356c1 cheers

File: backend/blueprint/api/controllers/assetHelper.py (hunk 1 above)
Comment (vaibhavsingh97) [reply backend/blueprint/api/controllers/assetHelper.py]: This PR strictly follows http://jsonapi.org/format/

File: backend/blueprint/api/utils/response.py [hunk 2]
```diff
@@ -9,12 +9,24 @@ def __init__(self, statusCode):
         super(Response, self).__init__()
         self.status = statusCode
 
-    def generateMessage(self, message):
+    def generateMessage(self, message, message_type, *args, **kwargs):
         self.id = uuid.uuid4()
-        self.attributes = message
-        self.type = 'imgResponse'
+        self.type = message_type
+        if isinstance(message, dict):
```
[the remark this answers is not in the document]
Comment (vaibhavsingh97) [reply backend/blueprint/api/utils/response.py]: We need this PR to handle response of https://github.com/fossasia/badgeyay/pull/785

**Comment by yashLadha** [comment]
As said there should be a single code convention to be followed in the
project and it seems you haven't seen the open event server codebase as
mentioned in my review. So let me explain you. If you go with the approach
you mentioned them we have to serialize the objects by a secondary logic
and which is inefficient. Using serializing library like marshmallow and
marshmallow_json API we don't have to take care of these things as done in
the open event server. The admin management can also be extended using the
library if you follow more closely and I don't think we need to maintain
this class any further as responses are already dumping form the library.
Also going with your case we have to at least prepare dicta to be passed in
class and that is cumbersome if we are going to extend the functionality.
So again it is my humble request to you that you please atleast go through
the codebase of open event server.

**Pull request closed without merging**